README.md

```diff
@@ -8,28 +8,20 @@
 $ skywire-updater -h
 
 skywire-updater is responsible for checking for updates, and updating services
-associated with skywire.
-
-It takes one optional argument [config-path] which specifies the path to the
-configuration file to use. If no [config-path] is specified, the following 
-directories are searched in order:
-
-  1. /Users/anonymous/.skycoin/skywire-updater/config.yml
-  2. /usr/local/skycoin/skywire-updater/config.yml
+associated with skywire. Services to be updated will be based on a specified configuration file.
 
 Usage:
-  skywire-updater [config-path] [flags]
```

Comment:
I would prefer it if we read the `[config-file]` via a CLI flag, instead of a new sub-command.  😜

Reply:
Thank you! I was thinking to use CLI flags originally too; I will go back and review the code again and see if I could make this change. Thank you!

Reply:
Hi Evan. I made a change to the code. It can now reads in [config-file] via CLI flag instead of having to type a new Cobra subcommand. It turns out that by default Cobra only parses local flags on the target command, any local flags on parent commands are ignored. So by enabling this argument Command.TraverseChildren allows Cobra to parse local flags on each command before executing the target command. Please let me know if this logic is okay. Thanks!